front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -138,7 +137,10 @@ const StdcmResults = ({
       const response = await postSimilarTrains({ body: request });
       const rawSimilarTrains = response.data?.similar_trains ?? [];
 
-      const enrichedSimilarTrains = addSecondaryCodesToSimilarTrains(rawSimilarTrains, pathSteps);
+      const enrichedSimilarTrains = addSecondaryCodesToSimilarTrains(
+        rawSimilarTrains,
+        (selectedSimulation.outputs as StdcmResultsOutput).pathProperties.manchetteOperationalPoints
```

Comment:
Why do we need a cast here? Ah, we do something similar above :/

Would it help to use `outputs` instead of `selectedSimulation.outputs as StdcmResultsOutput`?